File: Java ejs/UD3/A02/UD03A026.java
import java.util.Scanner;
public class UD03A026{
	public static void main(String[] args){
		int h,m,s;
        Scanner teclado=new Scanner(System.in);
        System.out.println("Introduzca horas: ");
        h=teclado.nextInt();
        System.out.println("Introduzca minutos: ");
        m=teclado.nextInt();
        System.out.println("Introduzca segundos: ");
        s=teclado.nextInt();
        if (h>=0 && h>24 && m>=0 && m<=59 && s>=0 && s<=59)
        {
            System.out.println("Hora correcta");
        }else{
            System.out.println("Hora incorrecta");
        }
	}
}

File: Python ejs/UD03/A02/UD03A026.py
h=int(input("Introduzca horas: "))
m=int(input("Introduzca minutos: "))
s=int(input("Introduzca segundos: "))
if h>=0 and h>24 and m>=0 and m<=59 and s>=0 and s<=59:
    print("Hora correcta")
else:
    print("Hora incorrecta")